Web UI/CognitiveSearch.UI/appsettings.json

```diff
@@ -1,41 +1,24 @@
-﻿{
+{
   "Logging": {
     "IncludeScopes": false,
     "LogLevel": {
       "Default": "Warning"
     }
   },
-
-    // Required fields
```

Comment:
The PR looks good to me, thanks for making the changes.
However I am still seeing the creds, not sure if git filter worked correctly or this is expected?